src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -393,3 +393,4 @@ class Realization:
     run_arg: "RunArg"
     num_cpu: int
     job_script: str
+    realization_memory: int  # Memory to reserve/book, in bytes
```

Comment:
Can we write this in a way so that we can drop the comment? Maybe we can include the unit in the property name?`reserved_realization_memory_in_bytes` maybe?

Reply:
I would claim that bytes is the canonical unit when the type is `int`, so the comment about `bytes` can strictly be dropped, but not sure if it makes it better. Since bytes is the canonical unit, it should not be necessary to have it in the variable names, so that puts me into a spagat :)